### PR TITLE
doc(README): add instructions on installing

### LINKED ISSUE #9: Fix the README install language
Currently it says:

```
Currently the only way to use the go version of the deis client is to build it yourself. To build the deis client, you need to have go, glide, and make installed. Then run make build.
```

This isn't true. After #7 is done, this repo will be publishing it to bintray (although deis/workflow was already publishing it), so the README should say that

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #9

relies on #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,50 @@
...
- and [make](https://www.gnu.org/software/make/) installed. Then run `make build`.
+### From Bintray
+
+Install the `deis` client from [bintray](https://bintray.com/) by running
+
+	$ curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash
+
+The installer will fetch the latest version of the client into your current directory.
+
+	$ ./deis --version
```
Comment (arschles) [inline README.md]: @bacongobbler maybe add some language here about moving the binary into your `PATH`?

**Comment by arschles** [comment]
@bacongobbler #8 didn't fix bintray deployment (I had to merge it to get Travis to try and deploy it, because AFAIK that's the only way to test the deploy, I digress), so we should probably hold off on merging this. Current status (as you probably know):

```console
ENG000656:workflow-cli aaronschlesinger$ curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash
Downloading deis-0.0.0-darwin-amd64 from Bintray...

deis is now available in your current directory.

To learn more about deis, execute:

    $ ./deis --help

ENG000656:workflow-cli aaronschlesinger$ ./deis
./deis: line 1: The: command not found
ENG000656:workflow-cli aaronschlesinger$ cat ./deis
The requested path was not found.ENG000656:workflow-cli aaronschlesinger$ 
```

Also, should we consider using a http://deis.io/deis-cli/install-v2-beta.sh script?

**Comment by arschles** [comment]
Requires #14

**Comment by bacongobbler** [comment]
> Also, should we consider using a http://deis.io/deis-cli/install-v2-beta.sh script?

That script does not exist. I'll file a PR to make one

**Comment by arschles** [comment]
@bacongobbler yea, sorry I wasn't too clear there and should have said exactly what you just did. Thanks!

File: README.md [hunk 2]
```diff
@@ -2,28 +2,50 @@
...
 ## Installation
 
-Currently the only way to use the go version of the `deis` client is to build it yourself.
-To build the `deis` client, you need to have [go](https://golang.org/), [glide](https://github.com/Masterminds/glide),
- and [make](https://www.gnu.org/software/make/) installed. Then run `make build`.
+### From Bintray
+
+Install the `deis` client from [bintray](https://bintray.com/) by running
+
+	$ curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash
```
Comment (jchauncey) [inline README.md]: @sgoings how hard would it be to update this script name?

Comment (sgoings) [inline README.md]: Lives here: https://github.com/deis/deis.io/tree/gh-pages/deis-cli

**Comment by arschles** [comment]
As of https://travis-ci.org/deis/workflow-cli/builds/114947501, the CLI is being uploaded properly to bintray. This LGTM after the script name is changed

**Comment by jchauncey** [comment]
once you merge the other PR to have a more generic v2 install script we should merge this.

**Comment by bacongobbler** [comment]
rebased and doc changed to reference install-v2.sh.

**Pull request merged**